README.md

```diff
@@ -104,6 +105,17 @@ Capture output from the container to the host running the command
         -final_delay 500 -c:v gif -f gif -ss 00:49:42 -t 5 - > trow_ball.gif
 ```
 
+#### Convert 10bits MKV into a 10Bits MP4
```

Comment:
ffmpeg is compiled with 8bit, 10bits and 12bits support, to use it pass the pix_fmt option